src/js/components/organisms/page-header/page-header.vue.html

```diff
@@ -1,18 +1,23 @@
+<!-- @format -->
+
 <header class="header" :class="headerClasses" role="banner" v-size-emitter="'header'">
     <div class="header__top" :class="{'header--mobile-inverted': mainNavOpen}">
         <router-link to="/" class="home-link" aria-label="Ultimaker homepage">
             <icon class="home-link__logo" icon-name="logo-ultimaker"></icon>
         </router-link>
 
-        <main-navigation v-if="navigation" :navigation="navigation" :main-nav-open="mainNavOpen" v-on="$listeners"></main-navigation>
+        <main-navigation v-if="navigation" :navigation="navigation" :main-nav-open="mainNavOpen" v-on="$listeners" />
 
+        <!--
```

Comment:
This will remove the order now from the nav. Comment the search out of the sys-nav component.